Libraries/IntegrationServiceLibrary.psm1

```diff
@@ -298,12 +298,12 @@ function Wait-ForVMToStartKVP {
 	param (
 		[String] $VmName,
 		[String] $HvServer,
-		[int] $StepTimeout
+		[int] $StepTimeout = 10
```

Comment:
Note that Wait-ForVMToStartKVP is being used in scripts with custom timeout values.
https://github.com/LIS/LISAv2/search?p=2&q=Wait-ForVMToStartKVP&unscoped_q=Wait-ForVMToStartKVP

So the stepTimeout should be enforced to 10 only if not specified to a custom value to come from other scripts calling this function.

Reply:
The Wait-ForVMToStartKVP is called by Get-IPv4AndWaitForSSHStart with a big time-out value, e.g, 1000. It doesn't make sense.

Reply:
@chvalean $StepTimeout = 10, it is a default parameter, means - if specify the value, then we use specified value, if not specified, then use 10

But here
https://github.com/LIS/LISAv2/pull/276/files#diff-41aa69e5e5d2928aef2eba282749a87fR306 we limit the max wait time is 5 mins, let know any concern, maybe not update code here, but update in each invoked places?